gateway/assets/dag.html

```diff
@@ -28,6 +28,33 @@
         <li><a href="?format=dag-cbor" rel="nofollow">Valid DAG-CBOR</a> (specs at <a href="https://ipld.io/specs/codecs/dag-cbor/spec/" target="_blank" rel="noopener noreferrer">IPLD</a> and <a href="https://www.iana.org/assignments/media-types/application/vnd.ipld.dag-cbor" target="_blank" rel="noopener noreferrer">IANA</a>)</li>
       </ul>
     </section>
+    {{ with .Node }}
+      <section class="full-width">
+        <header>
+          <strong>DAG Preview</strong>
```

Comment:
nit: this is not a DAG :)  lets use codec name here
```suggestion
          <strong><span translate="no" style="text-transform: uppercase;">{{.CodecName}}</span> preview</strong>
```